providers/fcm/src/lib/fcm.provider.ts

```diff
@@ -101,7 +101,9 @@ export class FcmPushProvider implements IPushProvider {
     await deleteApp(app);
 
     return {
-      ids: res?.responses?.map((response) => response.messageId),
+      ids: res?.responses?.map((response) =>
+        response.success ? response.messageId : response.error.message
```

Comment:
Does `response.error.message` return an id? How would we know to which id belongs this error?

Reply:
That was part of my point. Think of the scenario of 3 token, 2 of them are invalid. Does the error message provides the token identifier that error is related to or is just a generic error message? If it is the second case makes things more difficult to user because would need to debug in trial and error mode. Let me know if I didn't explain my worry clearly.

Reply:
In case of 3 invalid tokens -> ["invalid_token", "valid_token", "invalid_token"]
 
 execution log will be -> "ids": ["the registration token is not a valid FCM registration token", "projects/project-name/mesages/{messageId}", "the registration token is not a valid FCM registration token"]
 
 log will be in sync with token state
 Shall I add invalid token also in log?

Reply:
That's what I was thinking but was wondering if there was any way to get something like:
```
const tokens = ["invalid_token_1", "valid_token", "invalid_token_2"];

...

{
...
  "ids": ["the registration token invalid_token_1 is not a valid FCM registration token", "projects/project-name/mesages/{messageId}", "the registration token invalid_token_2 is not a valid FCM registration token"]
 ...
 }
 ```
Just to make life easier to the user. Not a big deal to not approve the PR on my side but if we could somehow show to the user which ones are invalid would be top.